schemas/BisSchemas/BisSchemas.PartFile.xml

```diff
@@ -600,7 +600,7 @@
     </Part>
 
     <Part Name="SchemaUpgradeCustomAttributes" BentleyBuildMakeFile="findSchemaPath.mki" BentleyBuildMakeOptions="-dSYMLINK_OUT=$(BuildContext)ECSchemas/Standard/SchemaUpgradeCustomAttributes.01.00.00.ecschema.xml -dSCHEMA_PATTERN=SchemaUpgradeCustomAttributes.*ecschema.xml"/>
-    <Part Name="CoreCustomAttributes" BentleyBuildMakeFile="findSchemaPath.mki" BentleyBuildMakeOptions="-dSYMLINK_OUT=$(BuildContext)ECSchemas/Standard/CoreCustomAttributes.01.00.03.ecschema.xml -dSCHEMA_PATTERN=CoreCustomAttributes.*ecschema.xml"/>
+    <Part Name="CoreCustomAttributes" BentleyBuildMakeFile="findSchemaPath.mki" BentleyBuildMakeOptions="-dSYMLINK_OUT=$(BuildContext)ECSchemas/Standard/CoreCustomAttributes.01.00.04.ecschema.xml -dSCHEMA_PATTERN=CoreCustomAttributes.*ecschema.xml"/>

```

Comment:
I think this change will break the `imodel-native` build as this schema is not present at the moment.

Reply:
The `imodel-native` build will not be affected by this change once the CoreCustomAttributes EC3.2 schema is released. The build pipelines will work fine with the latest schema.

Reply:
yes, we have to first complete the PR in `bis-schemas` repository. I will double check if the changes in both PR's are compatible with each other.

Reply:
`bis-schemas` [PR](https://github.com/iTwin/bis-schemas/pull/246) is now complete and the CoreCustomAttributes schema version `01.00.04` is released so the builds should pass now.